src/Analysis/GWASV2/Steps/ConfigureGWAS/ConfigureGWAS.stories.jsx

```diff
@@ -0,0 +1,179 @@
+            className='GWASUI-navBtn GWASUI-navBtn__next'
+            type='primary'
+            onClick={() => {
+              dispatch({ type: ACTIONS.INCREMENT_CURRENT_STEP });
+            }}
+            disabled={!nextButtonEnabled}
+          >
+            Next
+          </Button>
+        </div>
+      </SourceContextProvider>
+    </QueryClientProvider>
+  );
+};
+
+let rowCount = 0;
+
+export const MockedSuccess = MockTemplate.bind({});
+MockedSuccess.parameters = {
+  msw: {
+    handlers: [
+      rest.post(
+        'http://:cohortmiddlewarepath/cohort-middleware/concept-stats/by-source-id/:sourceid/by-cohort-definition-id/:cohortdefinition/breakdown-by-concept-id/:breakdownconceptid',
+        (req, res, ctx) => {
+          const { cohortmiddlewarepath } = req.params;
+          const { cohortdefinition } = req.params;
+          rowCount++;
+          if (rowCount == 12) {
+            // simulate empty response scenario:
+            return res(
+              ctx.delay(200 * rowCount),
+              ctx.json({
+                concept_breakdown: null,
+              })
+            );
+          }
+          return res(
+            ctx.delay(200 * rowCount),
+            ctx.json({
+              concept_breakdown: [
+                {
+                  concept_value: 'ASN',
+                  concept_value_as_concept_id: 2000007029,
+                  concept_value_name: 'non-Hispanic Asian',
+                  persons_in_cohort_with_value: 40178 * (20 - rowCount), // just to mock/generate different numbers for different cohorts,
+                },
+                {
+                  concept_value: 'EUR',
+                  concept_value_as_concept_id: 2000007031,
+                  concept_value_name: 'non-Hispanic White',
+                  persons_in_cohort_with_value: 39648 * (20 - rowCount), // just to mock/generate different numbers for different cohorts,
+                },
+                {
+                  concept_value: 'AFR',
+                  concept_value_as_concept_id: 2000007030,
+                  concept_value_name: 'non-Hispanic Black',
+                  persons_in_cohort_with_value: 40107 * (20 - rowCount), // just to mock/generate different numbers for different cohorts,
+                },
+                {
+                  concept_value: 'HIS',
+                  concept_value_as_concept_id: 2000007028,
+                  concept_value_name: 'Hispanic',
+                  persons_in_cohort_with_value: 40038 * (20 - rowCount), // just to mock/generate different numbers for different cohorts,
+                },
+              ],
+            })
+          );
+        }
+      ),
+
+      rest.post('http://:serverpath/ga4gh/wes/v2/submit', (req, res, ctx) => {
+        const { serverpath } = req.params;
+        console.log(serverpath);
+        return res(
+          ctx.delay(1100),
+          ctx.text("gwas-workflow-123456")
+        );
+      }),
+
+    ],
+  },
+};
+
+// TODO
```

Comment:
follow-up:
- make story for mocking submission error and showing error banner

Reply:
See ticket: https://ctds-planx.atlassian.net/jira/software/projects/VADC/boards/132/backlog?selectedIssue=VADC-374